assertj-core/src/main/java/org/assertj/core/api/AssertionsForClassTypes.java

```diff
@@ -897,13 +897,15 @@ public static NotThrownAssert assertThatNoException() {
    *
    * @param shouldRaiseThrowable The lambda with the code that should raise the exception.
    * @return The captured exception or <code>null</code> if none was raised by the callable.
-   * @see AssertionsForClassTypes#catchThrowableOfType(ThrowableAssert.ThrowingCallable, Class)
+   * @see AssertionsForClassTypes#catchThrowableOfType(Class, ThrowableAssert.ThrowingCallable)
    */
   public static Throwable catchThrowable(ThrowingCallable shouldRaiseThrowable) {
     return ThrowableAssert.catchThrowable(shouldRaiseThrowable);
   }
 
   /**
+   * @deprecated use {@link #catchThrowableOfType(Class, ThrowingCallable)} instead.
```

Comment:
Please move deprecated javadoc after the documentation. The doc should not start with `@deprecated`

Reply:
Indeed, `@deprecated` should be in the last position according to [Order of Tags](https://www.oracle.com/technical-resources/articles/java/javadoc-tool.html#orderoftags) from Oracle.

I'll fix it here although the order of the tags is generally not consistent in the codebase.

Is there anything between the `maven-javadoc-plugin` and Spotless that could help enforce an order in line with Oracle guidelines?

Reply:
Raised #3469.